docs/book/src/advanced/traits.md

```diff
@@ -58,6 +58,39 @@ impl Ord for u64 {
 
 To require a supertrait, add a `:` after the trait name and then list the traits you would like to require and separate them with a `+`.
 
+### ABI supertraits
```

Comment:
For the code snippets below, I suggest we add an actual example in https://github.com/FuelLabs/sway/tree/master/examples which can just be a single Sway file. We should probably avoid using the `Ownable` example just yet, since it's a bit misleading given that `only_owner()` here is not available externally.